backend/community/guides/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from guides.models import Guide
+
+
+@admin.register(Guide)
+class GuideAdmin(admin.ModelAdmin):
```

Comment:
I don't think we need this `ModelAdmin`. It's only useful if you want to provide some custom interface in the admin panel. Quoting django documentation:

"

Do you need a ModelAdmin object at all?

In the preceding example, the ModelAdmin class doesn’t define any custom values (yet). As a result, the default admin interface will be provided. If you are happy with the default admin interface, you don’t need to define a ModelAdmin object at all – you can register the model class without providing a ModelAdmin description. The preceding example could be simplified to:
```
from django.contrib import admin
from myproject.myapp.models import Author

admin.site.register(Author)
```
"

Reply:
Well, it's the only model (as of now) in the whole `guides` application.
We might want to define some metadata on that model for admin panel.
Or remove the admin panel altogether.